src/components/about/mediaButtonLogos.svelte

```diff
@@ -0,0 +1,78 @@
+<script lang="ts">
+	import type { Logo } from '$domains';
+
+	// Creating a collection of logos
+	const logos: Logo[] = [
```

Comment:
Die Logos können dann auch in `static` ausgelagert werden

Reply:
ist ausgelagert